packages/bus-merge-binding-factory/lib/ActorMergeBindingFactory.ts

```diff
@@ -0,0 +1,45 @@
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { IActionContext } from '@comunica/types';
+
```

Comment:
This file needs some more docs on what each class and param does (have a look at other buses to see examples).